### DISCUSSION TITLE
Question about final project delivery

### DISCUSSION THREAD

**Discussion by OmarHawary21** [MuhammadSulaiman001/opengl-lab#32] [Q&A] [answered]
Is it ok for you if we continued working on the project after we upload it on google drive?
It is like we have extra 3 days why can't we work on the project after we upload it

**Comment by MuhammadSulaiman001** [comment, marked as answer]
Sure you can continue working on the project after uploading the material on Google Drive. We assume that the code won't change too much in 3 days (like, do you think your code will differ in more than 10-15%  in 3 days?).

**Reply by OmarHawary21** [reply]
No, I don't think it will change that much but the reason why I am asking is that we have the network fundamentals project on Monday so once we are done with that project all of our time will be for your project so it might change a little bit like we will add lighting and some models to it.
anyway, thank you for answering me and I hope that our project will amaze you.
